x-pack/plugins/siem/public/pages/network/ip_details.tsx

```diff
@@ -135,6 +136,32 @@ const IPDetailsComponent = pure<IPDetailsComponentProps>(
                       />
                     )}
                   </UsersQuery>
+                  <EuiSpacer size="s" />
```

Comment:
As per the `Hosts Page` that @stephmilovic made UI updates to recently, I think we just want a regular `<EuiSpacer />` here between each table. (This was my by bad as I added these as part of the `Users Table`, sorry bout that!)

Reply:
Sounds good, thanks for looking around to double check. I added this between each table.